### PR TITLE
added Humor API

### PR DESCRIPTION
**Pull request by ddsky**
also updated spoonacular API description

- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -564,6 +564,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Halo](https://developer.haloapi.com/) | Halo 5 and Halo Wars 2 Information | `apiKey` | Yes | Unknown |
 | [Hearthstone](http://hearthstoneapi.com/) | Hearthstone Cards Information | `X-Mashape-Key` | Yes | Unknown |
 | [Humble Bundle](https://rapidapi.com/Ziggoto/api/humble-bundle) | Humble Bundle's current bundles | `apiKey` | Yes | Unknown |
+| [Humor API](https://humorapi.com) | Humor, Jokes, and Memes | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:567]: Title should not end with "... API". Every entry is an API here!

```suggestion
| [Humor](https://humorapi.com) | Humor, Jokes, and Memes | `apiKey` | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @ddsky :)

Can you resolve this review, please?

**Comment by ddsky** [comment]
Thanks for the change, I agree and committed your suggestion :)

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for the update @ddsky ;)

**Pull request merged**